index.src.html

```diff
@@ -137,8 +140,72 @@
+
+: <dfn>Authenticator</dfn>
+:: A cryptographic device used by a <a>[WAC]</a> to (i) generate a <a>scoped credential</a> and register it with a <a>[RP]</a>, and (ii) subsequently used to cryptographically sign and return, in the form of an <a>WebAuthn Assertion</a>, a challenge and other data presented by a <a>[RP]</a> (in concert with the <a>[WAC]</a>) in order to effect authentication. Broadly, authenticators are of two types: <a>Embedded Authenticators</a> or <a>External Authenticators</a>.
+
+: <dfn>Authorization Gesture</dfn>
+:: Essentially the same as <a>user verification</a>.
+
+: <dfn>Ceremony</dfn>
+:: The concept of a ceremony [[Ceremony]] is an extension of the concept of a network protocol, with human nodes alongside computer nodes and with communication links that include UI, human-to-human communication and transfers of physical objects that carry data. What is out-of-band to a  protocol is in-band to a ceremony. In this specification, <a>Registration</a>, <a>Authentication</a>, and <a>user verification</a> are ceremonies. 
+    
+: <dfn>Client</dfn>
+:: See <a>Conforming User Agent</a>.
+
+: <dfn>Conforming User Agent</dfn>
+:: A user agent implementing, in conjunction with the underlying platform, the <a>Web Authentication API</a> and algorithms given in this specification, and handling communication between 
+    <a>Authenticators</a> and <a>[RPS]</a>.
+
+: <dfn>eTLD+1</dfn>
+:: Also known as a <em>Registered Domain</em> [[PSL]], an eTLD+1 is an <em>effective Top-Level Domain Name</em> (eTLD), plus the next domain name label, proceding from right to left. An eTLD is also known as a <em>public suffix</em> [[RFC7719]].
+
+: <dfn>Registration</dfn>
+:: The <a>ceremony</a> where a user, a <a>[RP]</a>, and the user's computing device(s) (containing at least one <a>authenticator</a>) work in concert to create a <a>scoped credential</a> and associate it with the user's <a>[RP]</a> account.  Note that this includes employing <a>user verification</a>.
+
+: <dfn>Relying Party Identifier</dfn>
+: <dfn>RP ID</dfn>
+:: A Relying Party Identifier is derived from a <a>[RP]</a>'s web origin's hostname by computing the hostname's <a>eTLD+1</a>. 
+    
+: <dfn>Scoped Credential</dfn>
+:: Generically, a credential is data one entity presents to another in order to authenticate the former's identity [[RFC4949]]. A <em><a>scoped credential</a></em>, as defined here, is one that is cryptographically bound to a <a>[RP]</a>'s <a>RP ID</a>. Per specification, it is conveyed--i.e., registered--to only the <a>[RP]</a> instigating the credential's creation (see: <a>Registration</a>). Subsequently, only that <a>[RP]</a>, as identified by it's <a>RP ID</a> is able to employ the <a>scoped credential</a> in <a>authentication</a> ceremonies. 
```

Comment:
Re: "a credential is data one entity presents to another" this is clearly not true for credentials as described in this spec since the credential is a key pair and that is never presented to anyone. Perhaps more accurate to say "a credential is data that can be used by one entity..."?